Readme.md

```diff
@@ -1 +1,59 @@
-This is for the code review of todo-app
+Note: This is for the code review of todo-app
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To-Do List:
+
+Additional description about the project and its features.
+-Phase 1. No features added yet.
+-Phase 2. Added CRUD features.
+-Phase 3. Added the functionality to tick mark completed tasks. Also added the functionality to remove all completed tasks.
+
+# Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+- You can clone this repo by typing on your terminal.<pre><code>git clone https://github.com/muneebulrehman/ToDo-list.git</code></pre>
+- Type <pre><code>ToDo-list</code></pre> to access the project on the terminal
+
+## Built With
+
+- Major languages
```

Comment:
I see you have an amazing list of technologies on your project and it will impress visitors if you were more specific about them.

I would suggest removing this line as you already have a clear list of technologies
https://github.com/muneebulrehman/todo-review/blob/582e368b6867ed8f4e7683e7bd3ffe52f964ef53/Readme.md?plain=1#L21

Reply:
Thanks for suggestion. I have implemented it.